frontend_build/src/parse_bundle_plugin.js

```diff
@@ -130,6 +131,10 @@ var parseBundlePlugin = function(data, base_dir) {
       __publicPath: JSON.stringify(publicPath),
     }),
     new extract$trs(data.locale_data_folder, data.name),
+    new LodashModuleReplacementPlugin({
+      'collections': true,
+      'paths': true
```

Comment:
I think the linting error is probably coming from a lack of a trailing `,` here.

The pre-commit hooks are meant to catch these kind of things when you commit on your local machine: http://kolibri.readthedocs.io/en/develop/dev/getting_started.html#pre-commit-install

To fix this issue for sure, run the following command in your kolibri directory:
`yarn run lint-js:fix frontend_build/src/parse_bundle_plugin.js`.

Reply:
(sorry, updated command)